contracts/bonding/BondingManager.sol

```diff
@@ -421,20 +417,23 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         Delegator storage del = delegators[_transcoder];
 
         if (penalty > del.bondedAmount) {
-            // Decrease transcoder's total stake by transcoder's stake
```

Comment:
The transcoder is being removed from the pools in this function call anyway so we don't need to update its total stake